### ISSUE TITLE
[BITV] 9.1.4.13/3.1 - In the responsive (mobile) view, when focusing on the menu button, a tooltip opens that cannot be closed with the ESC key. (5)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41862]
![image](https://github.com/nextcloud/server/assets/25978914/0eb7eb50-5217-4ec2-a99b-6792c5b252d4)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3f706f8e21-v3-n1

9.1.4.13/4.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3f706f8e21-v4-n1

9.1.4.13/5.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3f706f8e21-v5-n1

9.1.4.13/9.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3f706f8e21-v9-n1

9.1.4.13/11.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3f706f8e21-v11-n1

**Comment by ShGKme** [comment]
It is to be fixed by:
1. https://github.com/nextcloud-libraries/nextcloud-vue/pull/4864 (the button is not initially focused)
2. https://github.com/nextcloud-libraries/nextcloud-vue/pull/4869 (the navigation is not initially open)
3. https://github.com/nextcloud-libraries/nextcloud-vue/pull/4888 (replaces custom tooltip with a native title)

**Comment by ShGKme** [comment]
@JuliaKirschenheuter I'm sorry, I missed that you already assigned yourself to the issue ><
